### ISSUE TITLE
Improve HTMLImageElement.loading

### ISSUE THREAD

**Issue by quinnlangille** [microsoft/TypeScript#48658]
# lib Update Request

## Configuration Check

My compilation *target* is `ES2015` and my *lib* is `the default`.

## Missing / Incorrect Definition

HTMLImageElement.loading is currently a `string`, but the only valid values for it are `eager` and `lazy`

## Sample Code

While the string _works_, it allows for invalid values. ex:

```html
<img loading="random value" />
``` 
is valid, as well as:
```typescript
interface ImageWrapper {
  image: HTMLImageElement
}

const imageWrapper: ImageWrapper = { image: new HTMLImageElement() }
imageWrapper.image.loading = 'random value'
```

I would expect both of these cases to throw a warning at least on invalid/not useful value

## Documentation Link

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading

**Comment by quinnlangille** [comment]
If everyone is in favour of making this change, I can open a PR anytime

**Comment by MartinJohns** [comment]
https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/747

**Comment by jakebailey** [comment]
This was fixed in #47445.
